api/src/services/cereTokensService.js

```diff
@@ -1,24 +1,24 @@
-const { decimal, CERE_MAINNET } = require("../constants/config");
+const { blockchainNames, networkNames, decimals } = require("../config/blockchains");
 const cereNetworkService = require('./cereNetworkService');
 const ethNetworkService = require('./ethNetworkService');
 const { ETHEREUM_CERE_LOCKED_ADDRESSES } = process.env;
 
 async function getTotalSupplyInternal(network) {
   const totalSupply = await cereNetworkService.getTotalSupply(network);
-  return +totalSupply / 10 ** decimal;
+  return +totalSupply / 10 ** decimals[blockchainNames.CERE];
```

Comment:
We should use BigNumber here as well.

Reply:
I suggest using here `BigNumber` just for mathematical operations and return a string in the end.

Reply:
resolved